.pylintrc

```diff
@@ -3,7 +3,7 @@
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=
+extension-pkg-whitelist=numpy

```

Comment:
I guess this is necessary when using intellij idea?

Reply:
I am getting lint errors for module not found in the pylint tests, for module random.random not found in NumPy.